/*
class MultipleCatchBlock{
    static void Main(){
        try{
            int[] numbers = {1,2,3,4};
            Console.WriteLine(numbers[5]);
        }catch(Exception e){
            Console.WriteLine("Error: Array index is out of range");
        }catch(Exception e){
            Console.WriteLine("Error occured");
        }finally{
            Console.WriteLine("Program execution completed");
        }
    }
}
*/